test/docfx.Tests/Api.verified.cs

```diff
@@ -2057,6 +2053,16 @@ public static class Build
             public const string EmptyInputFiles = "EmptyInputFiles";
         }
     }
+    public static class SystemTextJsonUtility
```

Comment:
Can this be made internal?

Reply:
I've changed to internal with commit.(https://github.com/dotnet/docfx/pull/9832/commits/8a3be9460c9800988b2727032f947f7b672ec90e)

InternalVisibleTo attributes are temporary required to test both SystemTextJson/NewtonsoftJson.